tests/hawc/apps/animal/test_animal_serializer.py

```diff
@@ -118,3 +124,96 @@ def test_dose_group_failures(self):
         for data in datasets:
             serializer = DosingRegimeSerializer(data=data)
             assert serializer.is_valid() is False
+
+
+@pytest.mark.django_db
+class TestEndpointSerializer:
```

Comment:
I figured out a more direct way to test the serializer, so moved the api tests to the serializer. We may do the same with other tests too since that better isolates what we really want to test.